web/src/pages/[...slugs].tsx

```diff
@@ -2,13 +2,13 @@
   Catch all possible routes and redirect the user to the root slug.
   Any predefined route takes precedence so this won't override them.
 */
-import {useRouter} from "next/router";
-import {useEffect} from "react";
+import { useRouter } from "next/router";
+import { useEffect } from "react";
 
 export default function AnySlug() {
   const router = useRouter();
 
   useEffect(() => {
-    router.replace("/");
+    void router.replace("/");
```

Comment:
Is there a reason for this use of `void`? I don't see a return here so seems unnecessary

Reply:
Router operations are async and I'm not awaiting in this case. One of the ESLint rules we are using specify that you should use void to indicate that this is an async operation but you are ignoring the returned promise

Reply:
Hmm, never heard of that usecase before, but makes sense.